docs/universal_gates.md

```diff
@@ -17,12 +17,12 @@ nav_order: 3
 
 ## Introduction
 
-Universal gates are gates which can be used to implement all other gates.
+Universal gates are gates that can be used to implement all other gates.
 This is useful as manufacturers only need to produce 1 type of universal gate to be able to use all other gates.
 
 The universal gates are NOR and NAND.
-This page will show you how to implement AND, OR and NOT gates using universal gates.
-The AND, OR and NOT gates are basic gates that are commonly used and are very important.
+This page will show you how to implement AND, OR, and NOT gates using universal gates.
+The AND, OR, and NOT gates are basic gates that are commonly used and are very important.
```

Comment:
required ?

Reply:
Just added a comma before the conjunction to separate two independent clauses/words.